app/scripts/controllers/preferences-controller.ts

```diff
@@ -120,6 +120,9 @@ export type PreferencesControllerState = Omit<
   | 'privacyMode'
   | 'tokenSortConfig'
   | 'useMultiRpcMigration'
+  | 'dismissSmartAccountSuggestionEnabled'
```

Comment:
Added here: https://github.com/MetaMask/core/pull/5866

Already covered in extension by a property nested under `preferences`